UnitySDK/Assets/ML-Agents/Editor/Builder.cs

```diff
@@ -0,0 +1,14 @@
+﻿#if UNITY_CLOUD_BUILD
```

Comment:
Can we maybe put that into the BuilderUtils.cs file ?

Reply:
Do you mean 1) Put UNITY_CLOUD_BUILD into builderUtils also? or 2) Put these two scripts as one?

Reply:
I've put the UNITY_CLOUD_BUILD flag into the builderUtils also